data-import-from-s3/src/main/java/no/unit/nva/dataimport/DataImportHandler.java

```diff
@@ -50,26 +51,34 @@ public DataImportHandler(S3Driver s3Driver, AmazonDynamoDB dynamoClient) {
         this.dynamoClient = dynamoClient;
     }
 
-    public ImportResult[] importAllFilesFromFolder(Map<String, String> request) {
+    public List<ImportResult<FailedDynamoEntriesReport>> importAllFilesFromFolder(Map<String, String> request) {
```

Comment:
This handler will change a lot in the future. I could delete it but I am keeping it only for maintaining the functionality.